packages/react-devtools-shared/src/devtools/views/Profiler/CommitFlamegraphListItem.js

```diff
@@ -43,6 +46,15 @@ function CommitFlamegraphListItem({data, index, style}: Props) {
     [selectFiber],
   );
 
+  const handleMouseEnter = (nodeData: ChartNodeType) => {
+    const {id, name} = nodeData;
+    hoverFiber({id, name});
+  };
+
+  const handleMouseLeave = () => {
+    hoverFiber(null);
+  };
```

Comment:
Looks like this is same functions as in https://github.com/facebook/react/blob/0bd6e07a970fc711855b184ca222ee6d1897ade3/packages/react-devtools-shared/src/devtools/views/Profiler/CommitRankedListItem.js#L48

So for consistency use variant in https://github.com/facebook/react/blob/0bd6e07a970fc711855b184ca222ee6d1897ade3/packages/react-devtools-shared/src/devtools/views/Profiler/CommitRankedListItem.js#L48

Reply:
Thanks, I removed this useCallback too:)